frontend/public/components/app.tsx

```diff
@@ -329,7 +329,19 @@ const AppRouter = () => {
     <Router history={history}>
       <CompatRouter>
         <Routes>
-          <Route path={LOGIN_ERROR_PATH} Component={AuthenticationErrorPage} />
+          {/* Auth error page needs HelmetProvider context, so wrap it */}
+          <Route
+            path={LOGIN_ERROR_PATH}
+            element={
+              <HelmetProvider>
+                <Helmet
+                  titleTemplate={`%s · ${getBrandingDetails().productName}`}
```

Comment:
Would be good to DRY this up so we don't duplicate the template (see line 227).

Reply:
maybe we can move the `HelmetProvider` higher up so we won't need 2 of them?